src/pages/Contests/UpcomingContests.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useMemo } from "react"
+import { DateTime } from "luxon"
+import { Box } from "../../components/Box"
+import { Column } from "../../components/Layout"
+import { Text } from "../../components/Text"
+import { Title } from "../../components/Title"
+
+import styles from "./Contests.module.scss"
+import { commify } from "../../utils/units"
+import { Table, TBody, Td, Th, THead, Tr } from "../../components/Table/Table"
+import { Contest } from "../../hooks/api/contests"
+
+type Props = {
+  contests?: Contest[]
+  onContestClick?: (id: number) => void
+}
+
+export const UpcomingContests: React.FC<Props> = ({ contests, onContestClick }) => {
+  const upcomingContests = useMemo(() => contests?.filter((c) => c.status === "CREATED"), [contests])
+
+  if (!upcomingContests || upcomingContests.length === 0) return null
```

Comment:
I assume this line makes sure the table will not show up if there are no upcoming contests (sorry I'm noob)